README.md

```diff
@@ -11,7 +11,7 @@
 - Css
 - positioning
 - PSuedo Classes
-- Flexbox
+- used javascript classes
 
 Javascrpit
 
```

Comment:
- [ ] **Optional:** For a cleaner and more professional code base, don't leave a block's pseudocode within your code. Kindly remove the pseudocode, a brief description like the header is enough.

https://github.com/ibrahim777764/project-awesome-books/blob/4440973b44c0beaacde36c1324f63dfcb3009cf7/index.js#L79-L81